### PR TITLE
chore(projects): add some tests for project commands

### PR DESCRIPTION
**Pull request by ps-kwang**

### CODE REVIEW

**Review by chesterbr — COMMENTED** [review]
I understand the failing lint test will be fixed by #58, so 👍 if it does.

**Review by chesterbr — APPROVED** [review]
Preemptive ✅

File: test/project.test.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      await assertLogin(() => root.execute(["project", "list"]));
+    });
+
+    it("update should fail", async () => {
+      await assertLogin(() => root.execute(["project", "update", "moo-cows"]));
+    });
+
+    it("link should fail", async () => {
+      await assertLogin(() => root.execute(["project", "link"]));
+    });
```
Comment (ani-ps) [inline test/project.test.ts:59]: should `delete` be here as well?

Comment (kevmo314) [reply test/project.test.ts:59]: Added delete

**Pull request merged**